src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx

```diff
@@ -71,143 +59,115 @@ const PatientView = (props: {
-                                            <p>
-                                                {
-                                                    props.patient?.address?.[0]
-                                                        ?.line?.[0]
-                                                }
-                                            </p>
-                                        ) : (
-                                            <div className="unavailable-content">
-                                                Ikke oppgitt
-                                            </div>
-                                        )}
-                                    </div>
-                                    <div className="item-container">
-                                        <h4>
-                                            <MobileOutlined className="field-icon" />
-                                            Telefon:
-                                        </h4>
-                                        {props.patient?.telecom?.[0]?.value !==
+                                <div className="item-container">
+                                    <h4>Kjønn</h4>
+                                    <p>
+                                        {props.patient.gender !== 'male' &&
+                                        props.patient.gender !== 'female'
+                                            ? props.patient.gender
+                                                  .charAt(0)
+                                                  .toUpperCase() +
+                                              props.patient.gender.slice(1)
+                                            : props.patient.gender === 'male'
+                                            ? 'Mann'
+                                            : 'Kvinne'}
+                                    </p>
+                                </div>
+
+                                <div className="item-container">
+                                    <h4>Fødselsdato</h4>
+                                    <p>
+                                        {props.patient.birthDate !==
                                         undefined ? (
-                                            <p>
-                                                {
-                                                    props.patient?.telecom?.[0]
-                                                        ?.value
-                                                }
-                                            </p>
+                                            calcAge()
                                         ) : (
                                             <div className="unavailable-content">
                                                 Ikke oppgitt
                                             </div>
                                         )}
-                                    </div>
+                                    </p>
```

Comment:
I think wrapping the div in a p gives warnings

Reply:
Fixed